### PR TITLE
Bump eslint deps

### PR DESCRIPTION
**Pull request by Palid**

### CODE REVIEW

File: src/@types/PushRules.ts [hunk 1]
```diff
@@ -91,10 +91,10 @@ export interface ISenderNotificationPermissionCondition
...
 
 export type PushRuleCondition = IPushRuleCondition<string>
-    | IEventMatchCondition
-    | IContainsDisplayNameCondition
-    | IRoomMemberCountCondition
-    | ISenderNotificationPermissionCondition;
+| IEventMatchCondition
+| IContainsDisplayNameCondition
+| IRoomMemberCountCondition
+| ISenderNotificationPermissionCondition;
```
Comment (t3chguy) [inline src/@types/PushRules.ts:97]: can't say I agree with this indentation

File: src/client.ts [hunk 2]
```diff
@@ -7990,9 +7990,9 @@ export class MatrixClient extends EventEmitter {
         limit?: number,
         batch?: string,
     ): Promise<{
-        rooms: ISpaceSummaryRoom[];
-        events: ISpaceSummaryEvent[];
-    }> {
+            rooms: ISpaceSummaryRoom[];
+            events: ISpaceSummaryEvent[];
+        }> {
```
Comment (t3chguy) [inline src/client.ts:7995]: can't say I agree with this indentation at all

**Review by t3chguy — CHANGES_REQUESTED** [review]

**Comment by t3chguy** [comment]
Warrants some code style discussion

File: src/client.ts (hunk 2 above)
Comment (Palid) [reply src/client.ts:7995]: Do we just set it to 2 for objects? Turning it off entirely isn't good either.

**Pull request closed without merging**